lib/youtube-dl.js

```diff
@@ -163,25 +163,37 @@ function call(video, args1, args2, options, callback) {
  * @param {Array.<String>} data
  */
 function filterData(data) {
-  var format = data[data.length - 1].split(' - ');
+  var info = JSON.parse(data);
+  var format = info.format.split(' - ');
+  function formatDuration(duration) {
```

Comment:
Moved in https://github.com/fent/node-youtube-dl/commit/c309c242ba6661c19613812313503cf9626fc264